hack/prepare-for-integration-tests.sh

```diff
@@ -122,7 +122,7 @@ while (("$#")); do
     alternate_deploy_supervisor=$1
     shift
     ;;
-  -c | --alternate-deploy-concierge)
+  --alternate-deploy-concierge)
```

Comment:
`-c` was already the shortcut for `--clean`, so this was not working here. Also, `-p` did not make sense as a shortcut for `--alternate-deploy-supervisor` either.

These options are currently not used often, so I removed the shortcut options for these. We can add shortcuts if these become commonly used.

Reply:
I prefer the long form anyways, except for really common tools (think `tar` and `curl`)

Reply:
Agree on the long form being better.
Happy to drop the shorts. `-s` was used so i went with the next consonant in "supervisor" which was `-p`, but agree its not great.